components/eam/bld/configure

```diff
@@ -1387,9 +1387,27 @@ if (($chem_pkg ne 'none') || ($prog_species)) {
 if ($chem_pkg =~ '_mam3') {
     $chem_cppdefs = ' -DMODAL_AERO -DMODAL_AERO_3MODE ';
 } elsif ($chem_pkg =~ '_mam4_mom' || $chem_pkg =~ '_mam4_resus_mom' ) {
+#for turning on CLDERA stratospheric volcanic aerosol treatment
+#    $chem_cppdefs = ' -DMODAL_AERO -DMODAL_AERO_4MODE_MOM ';
+    if (!defined $opts{'cldera_strat_volc'} ) {
```

Comment:
This logical would be much clearer if you put the `if (defined $opts{'cldera_strat_volc'})` part first, rather than just using the negation. `if .. else` is easier to read than `if not .. else`.

Reply:
Reworked this section of configure. Now it should be easier to read.